canonical/juju-dashboard
# WD-5857 - Merge audit logs into main

## Done

- Merge the audit logs feature branch into main.

## QA

- Connect to staging.
- Check that you can see audit logs.
- Connect to JAAS and you shouldn't see logs.

## Details

https://warthogs.atlassian.net/browse/WD-5857
